src/command_modules/azure-cli-natgateway/azure/cli/command_modules/natgateway/tests/latest/test_natgateway_commands.py

```diff
@@ -0,0 +1,55 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import re
+
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+
+class NatGatewayScenarioTests(ScenarioTest):
+
+    @ResourceGroupPreparer(location='eastus2')
+    def test_natgateway(self, resource_group, resource_group_location):
+
+        self.kwargs.update({
+            'resource_group': resource_group,
```

Comment:
The kwargs dict already contains `'rg': resource_group` so you don't need to put it in yourself. Just FYI.